tests/test_main.cpp

```diff
@@ -242,7 +242,9 @@ struct GodotTestCaseListener : public doctest::IReporter {
 			physics_server_2d = PhysicsServer2DManager::get_singleton()->new_default_server();
 			physics_server_2d->init();
 
+			ERR_PRINT_OFF
 			navigation_server_3d = NavigationServer3DManager::new_default_server();
+			ERR_PRINT_ON
```

Comment:
Can you move this down one line and include the 2D server as well, else I need to change that line again when the 2D navmesh baking pr adds the NavigationServer2DDummy.